tests/app/dao/test_organisation_dao.py

```diff
@@ -420,6 +421,24 @@ def test_add_to_and_get_email_branding_pool_for_organisation(sample_organisation
     assert third_branding not in results
 
 
+def test_dao_add_email_branding_list_to_organisation_pool(sample_organisation):
```

Comment:
Optional: Should we add a test for what happens if you call this with a branding that is already in the pool? I know we generally don't expect it but could have the admin app with a page open for the first user user, a second user updates the pool, the first user submits their form

Reply:
This would fail with an `IntegrityError` if you try to add a branding that is already in the pool. When writing this I thought that was ok, since you can't add branding that is already in the pool through admin. When you click "Submit" on the form, it does another [check of what's in the branding pool](https://github.com/alphagov/notifications-admin/pull/4306/files#diff-11eecc50c648e6b1c5c13b38769c3e8b3135e392651c134664c6de99ee0de65eR455-R459) (i.e. the allowed options) on the POST request and does not let you submit the form if with something that's already in the pool. This catches the scenario where two people are using the branding form at the same time and person 2 tries to add a brand that person 1 has just added.

(The error shown on the form is not very user friendly since it comes straight from WTForms, but it does stop you from ever adding branding that is already in a pool and would be tricky to make nicer.)

Let me know if you think that's ok or not - I had thought it was given that form is platform admin only and the chances of two people using it at the same time, let alone adding the same branding to the same pool are really, really low.

Reply:
A perfect explanation. I think that is definitely OK. Thank you for explaining for me. 

🚢